ui/imports/shared/popups/walletconnect/panels/ContentPanel.qml

```diff
@@ -0,0 +1,50 @@
+import QtQuick 2.15
+import QtQuick.Layouts 1.15
+
+import StatusQ.Core 0.1
+import StatusQ.Core.Theme 0.1
+
+Rectangle {
+    id: contentPanelRect

```

Comment:
We have a convention to use `root` as the toplevel item id

Reply:
Fixed